sql/connect/server/src/test/scala/org/apache/spark/sql/connect/ml/MLBackendSuite.scala

```diff
@@ -57,6 +57,7 @@ class MLBackendSuite extends MLHelper {
   }
 
   test("ML backend: estimator works") {
+    org.apache.spark.ml.util.ReadWriteUtils.safeMLClassLoader = null
```

Comment:
I don't quite like this approach. We should have a test-only conf and only allow arbitrary class if `Utils.isTesting && conf_enabled`. Then we set this conf in this test suite.

Reply:
This is not a test conf issue.
this is issue that we need to reset `safeMLClassLoader` before each test run, otherwise the test will reuse the safeMLClassLoader generated by the previous test